graphics/opencv/Portfile

```diff
@@ -275,8 +275,8 @@ variant qt5 conflicts qt4 description {Build with Qt5 Backend support.} {
 
 variant java description {Add Java bindings.} {
     PortGroup               java 1.0
-    java.version            12
-    java.fallback           openjdk12
+    java.version            1.6+
+    java.fallback           openjdk11
```

Comment:
Can I suggest you add a comment here, pointing to the ticket, unless you want to be doing this potentially again in the future?

Reply:
Now added.